### DISCUSSION TITLE
Serverless Function has crashed.

### DISCUSSION THREAD

**Discussion by coopersully** [vercel/community#2155] [Help]
### Summary

I've created a Flask app that acts as an account management system, but it doesn't seem to deploy properly. I'm a bit new to using Vercel and attempting to figure out the issue. When testing in my home environment and with **vercel dev**, it works great, but continues to throw the following on production:

<img width="405" alt="image" src="https://user-images.githubusercontent.com/29717904/233743187-202d39ac-084d-4102-9df5-31daa1aa364b.png">

Here's a link to the repository, it's a pretty tiny project: [https://github.com/coopersully/altbox](https://github.com/coopersully/altbox)
Any help or suggestions would be appreciated.

**Comment by michihupf** [comment]
Try and see if you can get the error message. Check the logs.
